interfaces/mount/backend.go

```diff
@@ -79,7 +80,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, confinement interfaces.ConfinementO
 		return fmt.Errorf("cannot synchronize mount configuration files for snap %q: %s", snapName, err)
 	}
 	if err := UpdateSnapNamespace(snapName); err != nil {
```

Comment:
the changes needs matching unit tests in the package

Reply:
Mmm... the unit test passes...

Reply:
Or do you mean to match the tests if changing from "daemon" alone to "daemon + endure"?

Reply:
@pedronis Can you be more specific, please? I don't understand what do you mean...

Reply:
I searched for that message but didn't find it in any place, so it should be ok...

Reply:
it's not ok. Previously the code in the if stanza was simple, it was just reporting an error, now there is actual logic so it merits a unit test both to reconfirm the logic and to make clear that the logic is intentional. Anyway I just created https://github.com/snapcore/snapd/pull/12940 that should help having a baseline to add tests for this here.